demos/simple-menu.html

```diff
@@ -17,11 +17,13 @@
 <html>
```

Comment:
I made the menu shorter and the box taller so it accomodated it in more screen sizes.